src/VisualStudio/Core/Def/Implementation/Remote/RemoteHostClientServiceFactory.RemoteHostClientService.cs

```diff
@@ -19,22 +21,22 @@ namespace Microsoft.VisualStudio.LanguageServices.Remote
 {
     internal partial class RemoteHostClientServiceFactory
     {
-        public class RemoteHostClientService : ForegroundThreadAffinitizedObject, IRemoteHostClientService
+        public sealed class RemoteHostClientService : ForegroundThreadAffinitizedObject, IRemoteHostClientService
         {
             /// <summary>
             /// this hold onto last remoteHostClient to make debugging easier
             /// </summary>
-            private static Task<RemoteHostClient> s_lastRemoteClientTask;
+            private static Task<RemoteHostClient?>? s_lastRemoteClientTask;
```

Comment:
would it possible be simpler to assigned SpecializedTask.Default to this, to prevent needin to worry about hte outer null?

Reply:
No really. We don't worry about it anywhere.